app/utils/AddressHelper.scala

```diff
@@ -67,6 +67,20 @@ object AddressHelper {
         case s: String => s
         case Some(s)   => s
       }
+    createHtmlBlock(lines)
+
+  }
+
+  def formatAddressBlock(address: AddressResponse): HtmlContent = {
```

Comment:
Not quite sure what this does but can see it takes a model and returns some html so I'm wondering - could this function live in a view or a view component? Seems like it's presentation logic so I'm inclined to think it should be in the presentation layer?

I guess is there any reason why we couldn't or shouldn't pass the Address model to our "Is this the correct address" view and have it decide how it wanted to render it?

Reply:
Thats fair, i think i put it here because this helper seems to serve as a place for manipulating the multiple address types we have in various ways - i have said before that anything involving html should probably live as a component, i think there could be a couple bits to refactor out of that.

I can introduce a component here to start with :)